pkgs/applications/science/logic/coq/default.nix

```diff
@@ -7,8 +7,9 @@
 
 { stdenv, fetchFromGitHub, writeText, pkgconfig
 , ocamlPackages, ncurses
-, buildIde ? true
+, buildIde ? !stdenv.lib.versionAtLeast version "8.10"
```

Comment:
I suggest you add a comment explaining the reason. It is important given that users may be surprised that CoqIDE is not available. And this file being linked from https://nixos.org/nixos/packages.html?attr=coq_8_10&channel=nixpkgs-unstable&query=coq, it may be viewed by users more frequently than you would expect.

Reply:
Done.